### PR TITLE
[MODULAR] Nerfs the stupid stripper pole boxes to not come with free tools.

### PR DESCRIPTION
**Pull request by Iamgoofball**
i love getting half a toolbox and the stuff to make a cattle prod from inside the permabrig :)

i have extended this logic to the rest of the boxes in here

### CODE REVIEW

**Comment by thestubborn** [comment]
the russians are coming to murder you for touching their precious code

File: modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm [hunk 1]
```diff
@@ -50,11 +50,7 @@
 
 /obj/item/storage/box/milking_kit/PopulateContents()
 	var/static/items_inside = list(
-		/obj/item/milking_machine/constructionkit = 1,
-		/obj/item/reagent_containers/glass/beaker = 1,
```
Comment (thestubborn) [inline modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm:54]: tbh i think the rest is fine, just the tools are bad imo

Comment (Iamgoofball) [reply modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm:54]: beakers being available in permabrig means clever prison botanists can splash tons of shit onto guards if they know what they're doing, and an upgraded power cell means prisoners can make cattle prods

Comment (thestubborn) [reply modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm:54]: make it the shitty ass battery and a small beaker?

**Comment by MassacreDemigirl** [comment]
just let me fix the brig sexvend with my PR so that we dont have to take away half the sex toys to be easily accesible

File: modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm (hunk 1 above)
Comment (thestubborn) [reply modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm:54]: also, prisons with grinders start with a large beaker

I'm not too bothered personally

**Comment by thestubborn** [comment]
̶o̶r̶ ̶w̶e̶ ̶c̶o̶u̶l̶d̶ ̶j̶u̶s̶t̶ ̶r̶e̶m̶o̶v̶e̶ ̶t̶h̶e̶ ̶f̶e̶t̶i̶s̶h̶ ̶c̶o̶n̶t̶e̶n̶t̶ ̶f̶u̶l̶l̶y̶,̶ ̶l̶i̶k̶e̶ ̶c̶h̶e̶s̶h̶ ̶w̶a̶n̶t̶e̶d̶

...never happening
but, anyway, other than that good pr

**Comment by Gandalf2k15** [comment]
Free tools for that stuff is no boeno.

**Pull request merged**